Dockerfile

```diff
@@ -17,22 +17,25 @@ RUN curl -Ls -o get-pip.py https://bootstrap.pypa.io/get-pip.py && \
         python-Consul==0.4.7 \
         manta==2.5.0
 
-# get Containerpilot release
-RUN export CP_VERSION=2.0.1 &&\
-   curl -Lo /tmp/containerpilot.tar.gz \
-   https://github.com/joyent/containerpilot/releases/download/${CP_VERSION}/containerpilot-${CP_VERSION}.tar.gz && \
-   tar -xzf /tmp/containerpilot.tar.gz && \
-   mv /containerpilot /bin/
+# Add ContainerPilot and set its configuration file path
+ENV CONTAINERPILOT_VER 2.0.1
+ENV CONTAINERPILOT file:///etc/containerpilot.json
+RUN export CONTAINERPILOT_CHECKSUM=a4dd6bc001c82210b5c33ec2aa82d7ce83245154 \
+    && curl -Lso /tmp/containerpilot.tar.gz \
+        "https://github.com/joyent/containerpilot/releases/download/${CONTAINERPILOT_VER}/containerpilot-${CONTAINERPILOT_VER}.tar.gz" \
+    && echo "${CONTAINERPILOT_CHECKSUM}  /tmp/containerpilot.tar.gz" | sha1sum -c \
+    && tar zxf /tmp/containerpilot.tar.gz -C /usr/local/bin \
+    && rm /tmp/containerpilot.tar.gz
 
-# configure Containerpilot and MySQL
-COPY bin/* /bin/
+# configure ContainerPilot and MySQL
 COPY etc/* /etc/
+COPY bin/* /usr/local/bin/
 
 # override the parent entrypoint
 ENTRYPOINT []
 
 # use --console to get error logs to stderr
-CMD [ "/bin/containerpilot", \
+CMD [ "/usr/local/bin/containerpilot", \
```

Comment:
Any reason not to drop the `/usr/local/bin/` and just have `CMD [ "containerpilot", ...` and rely on `PATH`?